0009-Archivematica-original-design-principles.md

```diff
@@ -0,0 +1,49 @@
+2. The information packages ingested by the application will be moved from one micro-service to the next using the Unix pipeline pattern.
+3. Micro-service functionality will be provided by one or more of the open-source software utilities and applications bundled into the application. Where necessary, these will be supplemented by integration code written as Python scripts.
+4. The application will provide a graphical user interface so that the end user can determine and control the status of digital objects moving through the pipeline.
+5. The application will generate AIPs that are system-independent, self-documenting and self-contained. The AIPs will contain PDI that conforms to recognized standards and schemas, such as PREMIS, METS and Dublin Core.
+6. The file-based AIP is the canonical source of the preserved digital objects and metadata. The preservation application may be supported by databases and indexes that are derived and updated from this source, but these are not essential to long-term preservation of the AIP.
+7. The application will be designed to integrate with diverse external systems capable of serving as sources of digital objects to be ingested.
+8. The application will be designed to integrate with diverse storage systems and protocols for deposit and long-term preservation of AIPs.
+9. The application will be designed to integrate with diverse external systems capable of providing search, browse and display capabilities for Dissemination Information Packages (DIPs) generated by the application.
+
+
+## Links
+
+* https://wiki.archivematica.org/Overview
+* https://wiki.archivematica.org/Dashboard Original dashboard functional requirements
+* https://wiki.archivematica.org/AIP_structure Reason for quad directories
+* https://smartech.gatech.edu/handle/1853/28490 “Permanent Objects, Disposable Systems”, Abrams, Stephen; Cruse, Patricia; Kunze, John, 2009
+* https://escholarship.org/uc/item/5313h6k9 “An emergent micro-services approach to digital curation infrastructure”, Abrams, Stephen; Kunze, John; Loy, David, 2009
+* Artefactual ipres presentations:
+* ipres2010: https://drive.google.com/file/d/0Bw6XXtMa5JRpcHp1Q0ZSV0tqdU0/view?usp=sharing 
+* ipres2012: https://drive.google.com/open?id=0B2qNaVLxjZmKamJkVmVFd3l1cVE 
+* ipres2013: https://drive.google.com/open?id=0B5OYDF_LHu4RY0ViLUlOd0hKUDA 
```

Comment:
I had a look at https://phaidra.univie.ac.at/search#?page=1&pagesize=10&collection=o:340041 and it seems that only a select few IPRES proceedings have been broken down into individual papers. An alternative to hosting on GDrive that might be more sustainable in the future is to use our https://github.com/archivematica/archivematica-case-studies resources folder, what do you think?